app/src/main/java/ai/elimu/appstore/model/appsynchronization/AppStatusModel.java

```diff
@@ -0,0 +1,48 @@
+package ai.elimu.appstore.model.appsynchronization;
+
+import com.google.gson.annotations.SerializedName;
+
+public class AppStatusModel {
```

Comment:
Is this used for representing the `ApplicationStatus` enum?: https://github.com/elimu-ai/model/blob/master/src/main/java/ai/elimu/model/enums/admin/ApplicationStatus.java